packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/FieldGuide.js

```diff
@@ -26,35 +30,26 @@ function FieldGuide ({
   const ModalComponent = modalComponent
   const id = activeItemIndex ? `field-guide-item-${activeItemIndex}` : 'field-guide-menu'
   return (
-    <ModalComponent
-      {...modalProps}
-    >
-      <Box
-        className={className}
-        height={{ min: boxHeight }}
-        width={{ min: boxWidth }}
-      >
+    <ModalComponent {...modalProps}>
+      <AutoMinHeight className={className} pad={{ left: 'medium', right: 'medium', bottom: 'medium', top: '0' }}>
```

Comment:
This div is explicitly given the css rule `min-height: auto` because previously all of the modal content was inheriting `min-height: 0` which made the flexbox components behave unexpected.

I'm also setting left, bottom, and right padding here while the top padding is set on the item's title. This preserves the previous styling, but allows for building a sticky title inside the scrollable item content.